### DISCUSSION TITLE
Size of the Headings

### DISCUSSION THREAD

**Discussion by tavlean** [notaapp/talk#191] [Feature Requests] [answered]
1. **H3**, **H4**, **H5** & **H6** are the exact sizes.

2. An option to make **H1**, **H2**, and **H3** a little bigger would be nice.

**Comment by astoilkov** [comment, marked as answer]
Yes, we will do that. We will implement custom themes for the editor where you could specify custom sizes for each heading.

**Reply by Arbiter313** [reply]
Just wondering whether there was any update on this. I agree that it would be sensible to make H3 smaller than H2, but bigger than H4, H5 and H6. That seems to be the behaviour most widely adopted by other Markdown apps.

**Reply by mathewmitchell** [reply]
Weird. H1, H2, H3 are different sizes for me. H3 and H4 look very similar in size, but I believe H3 is slightly bigger.

**Reply by Arbiter313** [reply]
I have just checked again and mine seem exactly the same size, that is weird.

**Reply by astoilkov** [reply]
We've researched this. We've also asked users from other apps about how they feel about theming there. For now, it feels like a big investment of time.

I suspect that if we ask users if they want us to spend X amount on theming or X amount on 10 other features they need they would choose the latter.

Alternatively, we can do theming quickly and badly and then suffer throughout the lifetime of the product. However, this doesn't support our philosophy.

**Comment by rudolphmax** [comment]
For me, the different heading-sizes are also a big issue. For me, it dramatically decreases readability and scanability of my documents.But I also know that implementing the infrastructure needed for proper and elegant "theming" is difficult and extremely time-consuming.

So, I suggest adding an option for a "multiplier" that decides on the relationship between the sizes of heading levels. This is common practice in web design today. [This website](https://typescale.com/) illustrates what I mean.
The heading level (1, 2, 3, 4, ...) gets raised to the power of the selected "scale" (as they call it on the website). A single option could thereby control all heading sizes, saving on real-estate in the menu. Also, by providing preset options, you could keep control of the aesthetic aspects but still offer reasonable customizability.
I feel, this is a modern, adequately customizable, elegant, and not too time-consuming approach.

Though, when implementing "theming" later down the road, this might need to be rethought.

**Reply by astoilkov** [reply]
Elegant idea. We will look into it. Thanks!
